src/Components/Profile/ChangePasswordSuccess.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { useEffect } from "react";
+import { useHistory, } from 'react-router-dom'
+import Good from "../Assets/good.png"
+
+const ChangePasswordSuccess = () => {
+    const history = useHistory()
+    useEffect(() => {
+        setTimeout(() => {
+            history.push('/profile')
+        }, 8000);
+        // eslint-disable-next-line
```

Comment:
why did you disable next line. remove and add missing dependencies

Reply:
Ok sir